tox.ini

```diff
@@ -0,0 +1,27 @@
+# Tox (http://tox.testrun.org/) is a tool for running tests
+# in multiple virtualenvs. This configuration file will run the
+# test suite on all supported python versions. To use it, "pip install tox"
+# and then run "tox" from this directory.
+
+[tox]
+envlist = py27, py35, docs
+
+[testenv]
+commands =
+    coverage run -m nose
```

Comment:
This results in a .coverage data file in the project root, which gets overwritten between py27 and py35. Looking at some of these tricky conditionals, I've wondered about their coverage, and I don't have a good way to see.